### DISCUSSION TITLE
World command doesn't list all worlds/ can only use numbers for "extra" worlds

### DISCUSSION THREAD

**Discussion by espen96** [Zrips/CMI#2463] [Q&A]
**Question: **
so I noticed the World command doesn't list worlds, and you can't use the world name to go to a specific world, only numbers.

Is there a way to set the name of a word in a way, to make it easier to use?

**Comment by CloudeLecaw** [comment]
you can go to worls by writing the name of it

**Comment by mrfloris** [comment]
There's a placeholder as well `%cmi_server_worlds%`
